instrumentation/spring/README.md

```diff
@@ -23,7 +23,7 @@ Using the [spring project initializer](https://start.spring.io/), we will create
 Add the dependencies below to enable OpenTelemetry in `MainService` and `TimeService`. The Jaeger and LoggingExporter packages are recommended for exporting traces but are not required. As of May 2020, Jaeger, Zipkin, OTLP, and Logging exporters are supported by opentelemetry-java. Feel free to use whatever exporter you are most comfortable with.
 
 Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://search.maven.org/search?q=g:io.opentelemetry).
- - Minimum version: `0.11.0`
+ - Minimum version: `1.0.0`
```

Comment:
should we use `1.1.0` as it's the latest stable release?

Reply:
sure, why not 😄👍